test/unit/update.js

```diff
@@ -350,7 +350,55 @@ exports['controller.update'] = {
         // Update Tessel was not called because it was already up to date
         test.equal(this.update.callCount, 1);
         // It was provided the binaries
-        test.equal(this.update.calledWith(binaries), true);
+        test.equal(this.update.calledWith(opts, binaries), true);
+        // Then Tessel was closed
+        test.equal(this.tessel.closed, true);
+        // We closed all open Tessel connections
+        test.equal(this.closeTesselConnections.callCount, 1);
+        // We called the close function with an array
+        test.equal(Array.isArray(this.closeTesselConnections.args[0]), true);
```

Comment:
Great start here! For this test, use test.deepEqual to compare arg[0] with the array you expect it to be